Patches/Erin's Azaphrae/Scenarios_Azaphrae.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+
+   <Operation Class="PatchOperationFindMod">
+      <mods>
+         <li>Erin's Azaphrae</li>
+      </mods>
+      <match Class="PatchOperationSequence">
+         <operations>
+
```

Comment:
For the scenario, the armor items (helmet, flak pants and vest) will either need to be removed (like we do for the vanilla crashlanded scenario) or will need the material specified to prevent 'MakeThing' errors when the scenario starts. The former would be the quickest approach, but I don't know if you feel this race has some special need to start with some armor for some reason.

Reply:
I don't think they do, they have ability to do short flight so most of the time they will have advantage over battle even during early game

Reply:
will do something to solve it, fix incoming soon